hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegionServer.java

```diff
@@ -1922,6 +1926,14 @@ private void startServices() throws IOException {
       HConstants.DEFAULT_THREAD_WAKE_FREQUENCY);
       healthCheckChore = new HealthCheckChore(sleepTime, this, getConfiguration());
     }
+    // Executor status collect thread.
+    if (this.conf.getBoolean(HConstants.EXECUTOR_STATUS_COLLECTE_ENABLED,
+        HConstants.DEFAULT_EXECUTOR_STATUS_COLLECTE_ENABLED)) {
+      int sleepTime = this.conf.getInt(ExecutorStatusChore.WAKE_FREQ,
+          ExecutorStatusChore.DEFAULT_WAKE_FREQ);
+      executorStatusChore = new ExecutorStatusChore(sleepTime, this, this.getExecutorService(),
```

Comment:
Are there other executor services running in RS? This the only one?

Reply:
Not the only one inHRegionServer#startServices()

Reply:
Hi Stack, maybe I misunderstood it. I mead there is only one ExecutorService, but it will start many Executors.

Reply:
My question is if this patch will get metrics on all executors in the regionserver or just one set? If it does not get metrics on them all, should it? Thanks.

Reply:
So, we only need one chore to report on all executors?